pkg/config/config.go

```diff
@@ -103,8 +103,9 @@ type MicroshiftConfig struct {
 	BaseDomain       string        `json:"baseDomain"`
 	Cluster          ClusterConfig `json:"cluster"`
 
-	Ingress IngressConfig      `json:"-"`
-	Etcd    InternalEtcdConfig `json:"etcd"`
+	Ingress                       IngressConfig      `json:"-"`
+	Etcd                          InternalEtcdConfig `json:"etcd"`
+	DisableLoadBalancerController bool               `json:"disableLoadBalancerController"`
```

Comment:
We will want this to be a string field so we can use an enum.

Reply:
May I ask why we prefer an enum instead of a boolean here?

Reply:
Of coruse, I should have given more detail.

Our general API design guidelines have good background: https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#do-not-use-boolean-fields

The short version is that it's difficult to expand the meaning of the field in the future (Imagine what we would do if we start embedding MetalLB in MicroShift, for example.), it is difficult to change the default, and in our case it is difficult to tell if a user has provided a value or not because the zero value is a valid value. We could use a pointer field for that, but an enum is easier to work with.